src/user.js

```diff
@@ -20,20 +20,24 @@ class User {
     }
   }
 
-  decideToCook(recipe) {
-    this.recipesToCook.push(recipe)
-  }
-
-  filterRecipes(type) {
-    return this.favoriteRecipes.filter(recipe => recipe.type.includes(type))
+  addPantryIngredientNames(ingredientList) {
+    this.pantry.forEach(pantryItem => {
+      const foundItem = ingredientList.find(item => item.id === pantryItem.ingredient)
+      pantryItem.name = foundItem.name
+    })
   }
 
-  searchForRecipe(keyword) {
-    return this.favoriteRecipes.filter(recipe => {
-      return recipe.name.includes(keyword) ||
-             recipe.ingredients.includes(keyword)
+  alphabetizePantry() {
+    return this.pantry.sort(function(a, b) {
+      if (a.name > b.name) {
+        return 1
+  
+      } else if (a.name < b.name) {
+        return -1
+      }
     })
   }
+
```

Comment:
Also nit-picky but I don't think this line-break is useful since it's the end of the class.